js/performance.ts

```diff
@@ -24,6 +20,6 @@ export class Performance {
     assert(msg.Any.NowRes === baseRes.innerType());
     const res = new msg.NowRes();
     assert(baseRes.inner(res) != null);
-    return res.time().toFloat64() - this.timeOrigin;
+    return (res.seconds().toFloat64() * 1e9 + res.subsecNanos()) / 1000000;
```

Comment:
Should be

 res.seconds() * 1000 + res.subsecNanos / 1000

Reply:
updated